solidity/contracts/AttributeCheckpointFraud.sol

```diff
@@ -26,7 +27,8 @@ struct Attribution {
  * @title AttributeCheckpointFraud
  * @dev The AttributeCheckpointFraud contract is used to attribute fraud to a specific ECDSA checkpoint signer.
  */
-contract AttributeCheckpointFraud is Ownable {
+
+contract AttributeCheckpointFraud is Ownable, PackageVersioned {
```

Comment:
Wdyt of checking in the contract with the version and then just have a git hook enforcing no diff? I know its pollutes the git diff more, but it honestly feels kinda dirty to modify smart contract source code at runtime

Reply:
wdym by modify at runtime
its still before compile time, the file is checked in with the version...

Reply:
530f985ed3cc0b2f2d15718cd73777063917dea9